lib/web_ui/test/text_editing_test.dart

```diff
@@ -2579,7 +2578,9 @@ Future<void> testMain() async {
 
       final DomHTMLElement input = editingStrategy!.activeDomElement;
       expect(input.style.getPropertyValue('forced-color-adjust'), 'none');
-    // TODO(hterkelsen): https://github.com/flutter/flutter/issues/115327
+    // TODO(hterkelsen): Firefox does not support forced-color-adjust even
+    // though it supports forced-colors. Safari doesn't support forced-colors
+    // so this isn't a problem there.
```

Comment:
Should we still run the test and just alter expectations on different browsers? The prod code will run on all browsers, it just produces different results.